pkg/controller/clusterdeployment/clusterdeployment_controller_test.go

```diff
@@ -2524,8 +2574,6 @@ func testEmptyClusterDeployment() *hivev1.ClusterDeployment {
 }
 
 func testInstallConfigSecret() *corev1.Secret {
-	encodedIC := base64.StdEncoding.EncodeToString([]byte(testAWSIC))
-	log.Info(encodedIC)
```

Comment:
Doesn't look like it is related to these changes

Reply:
It's not it's from a recent PR I did, but doing a separate PR and CI cycle to remove two lines of logging feels worse to me than including it in a diff PR. WDYT?

Reply:
No worries - just wanted to make sure it was purposeful